sys/hoon.hoon

```diff
@@ -12651,16 +12651,16 @@
          ==                                            ::
 ++  desk  @tas                                          ::  ship desk case spur
 ++  cage  (cask vase)                                   ::  global metadata
-++  cask  |*(a/$-(* *) (pair mark a))                   ::  global data
```

Comment:
this looks like it should obviously be `mold`, not `gate`? (similarly with `++pair` calls and such)